portal-web/test/functional/com/liferay/portalweb/macros/ChangeList.macro

```diff
@@ -86,11 +86,27 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
-	macro publishPublication {
-		Click(locator1 = "ChangeList#CHANGE_LIST_INDICATOR_BUTTON");
+	macro prepareToPublish {
```

Comment:
Yes! Thanks for splitting up these macros. Since they are two separate screens, it's a good idea to split these steps up. In the future, we might want to do actions or make assertions prior to publishing.